From abc1234abc1234abc1234abc1234abc1234abc1234 Mon Sep 17 00:00:00 2001
From: Jane Dev <jane@example.com>
Date: Mon, 18 Aug 2026 10:00:00 -0400
Subject: [PATCH] Fix the off-by-one error (#42)

This commit fixes the off-by-one error in the
loop bounds.

Signed-off-by: Jane Dev <jane@example.com>
---
 src/foo.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/foo.cpp b/src/foo.cpp
index 1234567..abcdefg 100644
--- a/src/foo.cpp
+++ b/src/foo.cpp
@@ -1,3 +1,3 @@
-for (int i = 0; i <= n; ++i)
+for (int i = 0; i < n; ++i)
